addons/binding/org.openhab.binding.sleepiq/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,37 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: SleepIQ Binding
+Bundle-SymbolicName: org.openhab.binding.sleepiq;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.1.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-ClassPath: .,
+ lib/sleepiq-api.jar
+Import-Package: 
+ com.eclipsesource.jaxrs.provider.gson,
```

Comment:
Ok, so this was expected to happen: https://openhab.ci.cloudbees.com/job/openHAB-Distribution/955/console

I will fix it by adding the dependency to the SleepIQ karaf feature - but it would be cool, if @syphr42 could remove this dependency from the included jar!